src/port1.0/portbuild.tcl

```diff
@@ -86,8 +88,12 @@ options build.type.add_deps
 default build.type.add_deps yes
 
 proc portbuild::build_getmaketype {args} {
+    global xcodebuildcmd
     if {[option build.type] eq "default"} {
         return [findBinary make $portutil::autoconf::make_path]
+    }]
+    if {[option use_xcode] eq "yes" && $xcodebuildcmd eq "none"} {
+        return -code error "This port requires Xcode, which was not found on your system!"
```

Comment:
That's probably not a good place to do this. This will cause an error when users that do not have Xcode installed try to parse the Portfile, and the port will hence completely disappear from their system because `portindex` will fail to find the details of this port.

Instead, you should do this check in one of the phases, e.g. in a pre-fetch block.